AirportTicketBooking/Dtos/PassengerDto.cs

```diff
@@ -0,0 +1,10 @@
+namespace AirportTicketBooking.Dtos
+{
+    public record PassengerDto(
+        int Id,
+        string Name,
+        string Email,
+        string PassportNumber,
+        string CreditCardNumber
```

Comment:
you did the property names right here, do the name thing with the Entities

Reply:
Thank you! I corrected the property names.